synchronized/analysis_options.yaml

```diff
@@ -6,9 +6,9 @@ include: package:pedantic/analysis_options.yaml
 analyzer:
   enable-experiment:
     - non-nullable
-  strong-mode:
-    implicit-casts: false
-    # implicit-dynamic: false
+  # strong-mode:
```

Comment:
Note that this will still have one effect - which is to not allow implicit downcasts from `dynamic`. I would suggest leaving it on personally.

Reply:
Ah ok thanks, I thought I read once that this should be removed with nnbd (but could not find anymore, maybe I read it wrong). That is the option I prefer so far! I will revert. Thanks jake!

Reply:
reverted!